lib/mayaUsd/render/vp2RenderDelegate/basisCurves.cpp

```diff
@@ -1525,11 +1525,6 @@ void HdVP2BasisCurves::_UpdateDrawItem(
 */
 HdDirtyBits HdVP2BasisCurves::_PropagateDirtyBits(HdDirtyBits bits) const
 {
-    // Visibility and selection result in highlight changes:
```

Comment:
DirtySelection is gone, it was never really used properly. There is now only DirtySelectionHighlight and DirtySelectionMode.